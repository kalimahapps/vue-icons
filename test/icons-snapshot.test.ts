import { expect, test, describe } from 'vitest';
import { defineComponent } from 'vue';
import { mount } from '@vue/test-utils';
import {
	CaAccessibilityColorFilled,
	AdArpdownandup,
	AdHardclip,
	AdPresetAb,
	AdUsb,
	AiAcclaim,
	AiOrcid,
	AiScirateSquare,
	AkAirplayAudio,
	AkBackspace,
	AkBootstrapFill,
	AkDribbbleFill,
	AkMoon,
	AkRibbon,
	AkWallet
} from '@kalimahapps/vue-icons';

const imports = {
	CaAccessibilityColorFilled,
	AdArpdownandup,
	AdHardclip,
	AdPresetAb,
	AdUsb,
	AiAcclaim,
	AiOrcid,
	AiScirateSquare,
	AkAirplayAudio,
	AkBackspace,
	AkBootstrapFill,
	AkDribbbleFill,
	AkMoon,
	AkRibbon,
	AkWallet,
};

describe('Test sample icons from each package', () => {
	test.each(Object.entries(imports))('test %s snapshot', (name, icon) => {
		// Mount an instance and inspect the render output
		// eslint-disable-next-line vue/require-expose
		const wrapper = mount(defineComponent({
			props: {
				color: String,
			},

			setup() {
				return icon;
			},
		}));

		expect(wrapper.html()).toMatchSnapshot();
	});
});