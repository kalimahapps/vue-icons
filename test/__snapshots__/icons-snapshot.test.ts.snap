// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Test sample icons from each package > test AdArpdownandup snapshot 1`] = `
"<svg width="1em" height="1em" fill="currentColor" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 256 256">
  <path fill-rule="evenodd" d="M80 75.996A4.002 4.002 0 0 1 75.996 80H52.004A4.002 4.002 0 0 1 48 75.996V52.004A4.002 4.002 0 0 1 52.004 48h23.992A4.002 4.002 0 0 1 80 52.004v23.992zm16 64A4.002 4.002 0 0 1 91.996 144H68.004A4.002 4.002 0 0 1 64 139.996v-23.992A4.002 4.002 0 0 1 68.004 112h23.992A4.002 4.002 0 0 1 96 116.004v23.992zm16 64a4.002 4.002 0 0 1-4.004 4.004H84.004A4.002 4.002 0 0 1 80 203.996v-23.992A4.002 4.002 0 0 1 84.004 176h23.992a4.002 4.002 0 0 1 4.004 4.004v23.992zm64 0a4.002 4.002 0 0 1-4.004 4.004h-23.992a4.002 4.002 0 0 1-4.004-4.004v-23.992a4.002 4.002 0 0 1 4.004-4.004h23.992a4.002 4.002 0 0 1 4.004 4.004v23.992zm16-64a4.002 4.002 0 0 1-4.004 4.004h-23.992a4.002 4.002 0 0 1-4.004-4.004v-23.992a4.002 4.002 0 0 1 4.004-4.004h23.992a4.002 4.002 0 0 1 4.004 4.004v23.992zm16-64A4.002 4.002 0 0 1 203.996 80h-23.992A4.002 4.002 0 0 1 176 75.996V52.004A4.002 4.002 0 0 1 180.004 48h23.992A4.002 4.002 0 0 1 208 52.004v23.992z"></path>
</svg>"
`;

exports[`Test sample icons from each package > test AdHardclip snapshot 1`] = `
"<svg width="1em" height="1em" fill="currentColor" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 256 256">
  <path fill-rule="evenodd" d="m25.856 160.231-.105 15.5 21.52.091s10.258.899 17.47-1.033c7.21-1.932 5.846.283 11.266-6.664s2.59-5.662 5.685-15.063c3.095-9.402 5.482-18.859 5.482-18.859l5.383-19.944 6.906-17.103 4.976-5.127 11.477-.617 16.25.1 18.06-.211 6.094.464 5.18 7.82 4.468 14.117 4.062 14.727 4.04 14.208 4.367 14.726s2.14 7.77 6.398 11.62c4.257 3.851 13.406 6.293 13.406 6.293l20.313.3 13.651-.105.502-15.884-16.709.405-17.022-.72-2.563-.717-5.742-17.059-6.713-23.695-5.777-19.032s-1.753-7.91-6.973-13.517c-5.22-5.607-8.141-8.08-15.059-10.146-6.917-2.066-10.042-.902-21.245-.803-11.202.099-17.124.015-22.405.19-5.281.174-10.457.896-10.457.896l-9.33 3.96-8.1 11.07-5.023 12.188-5.23 18.891-3.999 14.727-4.511 13.608-3.282 9.445-17.84.793-18.87.16z"></path>
</svg>"
`;

exports[`Test sample icons from each package > test AdPresetAb snapshot 1`] = `
"<svg width="1em" height="1em" fill="currentColor" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 256 256">
  <path fill-rule="evenodd" d="M101 124.751c0-1.106.89-2.003 1.992-2.003h25.707L129 105l28 24.5-28.301 24.5.301-18.52h-26.009c-1.1 0-1.991-.895-1.991-2.003v-8.726zM89.047 161H77.262v-15.955h-41.43V161H24.048v-32.5c0-4.744.816-9.11 2.448-13.1 1.632-3.989 3.898-7.418 6.8-10.29 2.9-2.87 6.33-5.106 10.289-6.708C47.542 96.801 51.864 96 56.547 96H83.11a5.95 5.95 0 0 1 2.312.453 5.84 5.84 0 0 1 3.173 3.173 5.95 5.95 0 0 1 .453 2.312V161zm-53.214-27.74h41.43v-25.475H56.546c-.362 0-1.125.053-2.289.159-1.163.106-2.508.37-4.034.793-1.526.423-3.135 1.073-4.827 1.95-1.693.876-3.249 2.084-4.669 3.625-1.42 1.542-2.591 3.468-3.513 5.78-.922 2.311-1.382 5.114-1.382 8.408v4.76zm194.757 12.681c0 1.429-.246 2.96-.737 4.597-.49 1.637-1.294 3.154-2.41 4.553-1.115 1.398-2.566 2.566-4.351 3.503-1.785.937-3.972 1.406-6.56 1.406H172.48a5.974 5.974 0 0 1-2.32-.446 5.673 5.673 0 0 1-1.853-1.228 5.673 5.673 0 0 1-1.227-1.852 5.974 5.974 0 0 1-.447-2.32v-52.307a5.935 5.935 0 0 1 1.674-4.15 5.504 5.504 0 0 1 1.852-1.25A5.974 5.974 0 0 1 172.48 96h38.828c1.429 0 2.96.253 4.597.759 1.637.506 3.162 1.316 4.575 2.432 1.413 1.116 2.589 2.566 3.526 4.352.937 1.785 1.406 3.972 1.406 6.56v2.187c0 2.053-.343 4.262-1.027 6.628-.684 2.365-1.785 4.604-3.303 6.717 1.31.803 2.537 1.77 3.682 2.9a18.572 18.572 0 0 1 3.013 3.928c.863 1.488 1.547 3.169 2.053 5.043.506 1.875.759 3.943.759 6.204v2.231zm-11.604-2.231c0-1.518-.238-2.879-.714-4.084-.476-1.205-1.146-2.239-2.009-3.102-.862-.862-1.904-1.524-3.124-1.986-1.22-.46-2.588-.691-4.106-.691H182.88v-11.694h20.932c1.517 0 2.886-.23 4.105-.691 1.22-.462 2.262-1.124 3.125-1.986.862-.863 1.524-1.897 1.986-3.102.46-1.205.691-2.566.691-4.084v-2.187c0-1.666-.803-2.5-2.41-2.5h-33.07v40.793h38.292c.208 0 .461-.015.759-.045.297-.03.565-.119.803-.267.238-.15.447-.402.625-.759.179-.357.268-.848.268-1.473v-2.142z"></path>
</svg>"
`;

exports[`Test sample icons from each package > test AdUsb snapshot 1`] = `
"<svg width="1em" height="1em" fill="currentColor" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 256 256">
  <path fill-rule="evenodd" d="M129.042 230.413c14.807.56 24.14-9.993 24.14-20.644 0-10.652-10.127-19.1-15.966-19.1 0-6.462 0-34.093-.022-39.406 5.368-1.812 8.29-1.81 17.39-5.088 9.097-3.278 8.624-2.793 15.03-8.938 6.407-6.144 7.441-24.457 7.441-30.875 3.484-.18 13.856.036 13.856.036l-.422-33.11-37.406.393v32.078h11.25c0 3.828.86 12.614-1.518 18.645-2.377 6.031-2.992 5.485-9.732 8.245-6.74 2.759-15.94 4.323-15.94 4.323s-.36-73.992-.144-73.992l13.224.001c1.102 0 1.52-.767.942-1.7l-22.217-35.878-22.728 36.321c-.586.936-.17 1.676.943 1.651l15.465-.339s.75 97.485.732 101.874c-4.272 0-13.788-.045-20.224-5.307-6.436-5.263-7.105-8.643-8.932-14.428-1.828-5.785-2.512-12.533-2.512-17.572 6.495 0 17.021-10.184 17.021-19.98 0-9.798-11.035-22.247-21.027-22.247-9.993 0-23.23 8.282-23.23 20.986 0 12.705 10.022 21.24 15.59 21.24 0 3.907.606 13.36 2.102 19.03 1.496 5.673 2.405 5.92 3.447 8.371 2.277 5.357 2.599 9.327 13.277 15.643 10.678 6.315 20.318 6.101 24.488 6.101.25 2.888 0 10.361 0 13.922-7.875 0-17.2 7.9-17.2 19.29 0 11.387 8.075 19.894 22.882 20.454z"></path>
</svg>"
`;

exports[`Test sample icons from each package > test AiAcclaim snapshot 1`] = `
"<svg width="1em" height="1em" viewBox="0 0 384 512" fill="currentColor" xmlns="http://www.w3.org/2000/svg">
  <path d="M192 64C176.376 74.86 34.879 208.266 27.429 221.373v72.22c10.325.253 147.565-143.552 164.571-150.382 17.006 6.83 154.246 150.635 164.571 150.382v-72.22C349.121 208.266 207.624 74.859 192 64Zm0 154.406C176.376 229.266 34.879 362.672 27.429 375.78v72.22c10.325.254 147.565-143.551 164.571-150.383C209.006 304.448 346.246 448.253 356.571 448v-72.221C349.121 362.672 207.624 229.266 192 218.406Z" style="" stroke-width=".0671184"></path>
</svg>"
`;

exports[`Test sample icons from each package > test AiOrcid snapshot 1`] = `
"<svg width="1em" height="1em" viewBox="0 0 512 512" fill="currentColor" xmlns="http://www.w3.org/2000/svg">
  <path d="M336.62 194.538c-7.13-3.328-13.866-5.56-20.253-6.614-6.365-1.095-16.574-1.612-30.71-1.612h-36.704v152.747h37.634c14.673 0 26.081-1.013 34.224-3.017 8.142-2.004 14.921-4.526 20.356-7.626a69.448 69.448 0 0 0 14.942-11.388c14.488-14.714 21.742-33.273 21.742-55.717 0-22.052-7.44-40.052-22.341-53.982-5.498-5.166-11.822-9.444-18.89-12.793zM256 8C119.022 8 8 119.042 8 256s111.022 248 248 248 248-111.042 248-248S392.978 8 256 8Zm-82.336 357.513h-29.389V160.148h29.389zM158.95 138.696c-11.14 0-20.213-9.01-20.213-20.212 0-11.118 9.052-20.191 20.213-20.191 11.18 0 20.232 9.052 20.232 20.191a20.194 20.194 0 0 1-20.232 20.212zm241.386 163.597c-5.29 12.545-12.834 23.581-22.65 33.088-9.982 9.837-21.597 17.194-34.844 22.196-7.75 3.017-14.839 5.063-21.307 6.117-6.49 1.013-18.828 1.509-37.076 1.509h-64.956V160.148h69.233c27.962 0 50.034 4.154 66.32 12.545 16.265 8.37 29.181 20.728 38.792 36.972 9.61 16.265 14.425 34.018 14.425 53.196.023 13.765-2.666 26.908-7.936 39.432z" style="" stroke-width=".07717"></path>
</svg>"
`;

exports[`Test sample icons from each package > test AiScirateSquare snapshot 1`] = `
"<svg width="1em" height="1em" viewBox="0 0 448 512" fill="currentColor" xmlns="http://www.w3.org/2000/svg">
  <path d="M48 32C21.5 32 0 53.5 0 80v352c0 26.5 21.5 48 48 48h352c26.5 0 48-21.5 48-48V80c0-26.5-21.5-48-48-48H48zm112.498 101.084h127.004v7.473c-16.459.612-27.408 3.073-32.799 7.357-5.377 4.31-8.047 12.826-8.047 25.547v164.922c0 12.695 2.761 21.328 8.15 25.781 5.445 4.518 16.37 6.953 32.698 7.305l-.43 7.447H160.498v-7.447c16.757-.352 27.773-2.617 33.086-6.823 5.325-4.166 7.943-12.943 7.943-26.263V173.46c0-12.721-2.618-21.238-7.943-25.547-5.313-4.284-16.329-6.745-33.086-7.357v-7.473zm-76.902 8.865c17.409 0 29.882 4.885 37.357 14.676 7.5 9.765 11.225 26.575 11.225 50.469v35.078c0 25.677 4.295 43.241 12.838 52.682 5.169 5.715 13.269 9.714 23.763 12.449v12.916c-22.018-3.308-39.31-10.196-51.875-20.742-16.784-14.063-25.117-34.596-25.117-61.563v-43.54c0-17.71-1.51-29.063-4.57-34.024-3.047-4.935-8.543-7.383-16.434-7.383H64V143.5a97.745 97.745 0 0 1 9.063-1.172 144.62 144.62 0 0 1 10.533-.379zm280.808 0c3.815 0 7.304.144 10.534.38 3.229.273 6.236.663 9.062 1.171v9.467h-6.783c-7.89 0-13.387 2.448-16.434 7.383-3.06 4.96-4.57 16.315-4.57 34.023v43.541c0 26.967-8.333 47.5-25.117 61.563-12.566 10.546-29.857 17.434-51.875 20.742v-12.916c10.494-2.735 18.594-6.734 23.763-12.45 8.543-9.44 12.838-27.004 12.838-52.681v-35.078c0-23.894 3.725-40.704 11.225-50.469 7.475-9.79 19.948-14.676 37.357-14.676z"></path>
</svg>"
`;

exports[`Test sample icons from each package > test AkAirplayAudio snapshot 1`] = `
"<svg width="1em" height="1em" fill="none" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
  <path stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="m12 16 6 5H6l6-5Z"></path>
  <path stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M4 18a9.956 9.956 0 0 1-2-6C2 6.477 6.477 2 12 2s10 4.477 10 10a9.956 9.956 0 0 1-2 6"></path>
  <path stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M17.123 15.125a6 6 0 1 0-10.247-.002"></path>
  <path stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M14 12a2 2 0 1 0-4 0"></path>
</svg>"
`;

exports[`Test sample icons from each package > test AkBackspace snapshot 1`] = `
"<svg width="1em" height="1em" fill="none" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
  <path stroke="currentColor" stroke-linecap="round" stroke-width="2" d="m17 15-6-6m6 0-6 6"></path>
  <path stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M7.4 4.8A2 2 0 0 1 9 4h11a2 2 0 0 1 2 2v12a2 2 0 0 1-2 2H9a2 2 0 0 1-1.6-.8l-4.5-6a2 2 0 0 1 0-2.4l4.5-6Z"></path>
</svg>"
`;

exports[`Test sample icons from each package > test AkBootstrapFill snapshot 1`] = `
"<svg width="1em" height="1em" fill="none" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
  <path fill="currentColor" fill-rule="evenodd" d="M4.985 2c-1.37 0-2.383 1.199-2.337 2.498.043 1.25-.013 2.867-.42 4.186-.41 1.322-1.1 2.16-2.228 2.268v1.215c1.128.107 1.819.945 2.227 2.268.408 1.319.464 2.936.42 4.185-.045 1.3.968 2.499 2.338 2.499h14.032c1.37 0 2.383-1.199 2.337-2.499-.043-1.249.013-2.866.42-4.185.409-1.323 1.098-2.16 2.226-2.268v-1.215c-1.128-.108-1.817-.946-2.226-2.268-.407-1.32-.463-2.937-.42-4.186C21.4 3.198 20.386 2 19.017 2H4.985ZM16.27 13.769c0 1.79-1.335 2.875-3.55 2.875H8.949a.407.407 0 0 1-.407-.407V6.881a.407.407 0 0 1 .407-.406h3.75c1.847 0 3.06 1 3.06 2.537 0 1.078-.816 2.043-1.855 2.213v.056c1.415.155 2.367 1.135 2.367 2.488ZM12.31 7.764h-2.15v3.038h1.811c1.4 0 2.172-.564 2.172-1.572 0-.944-.664-1.466-1.833-1.466Zm-2.15 4.243v3.347h2.23c1.457 0 2.23-.585 2.23-1.684 0-1.1-.794-1.663-2.324-1.663h-2.136Z" clip-rule="evenodd"></path>
</svg>"
`;

exports[`Test sample icons from each package > test AkDribbbleFill snapshot 1`] = `
"<svg width="1em" height="1em" fill="none" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
  <g clip-path="url(#a)">
    <path fill="currentColor" fill-rule="evenodd" d="M12 0C5.375 0 0 5.375 0 12s5.375 12 12 12c6.612 0 12-5.375 12-12S18.612 0 12 0Zm7.926 5.531a10.202 10.202 0 0 1 2.317 6.378c-.338-.065-3.722-.755-7.132-.325-.079-.17-.144-.352-.222-.534a30.53 30.53 0 0 0-.676-1.484c3.774-1.536 5.492-3.748 5.713-4.035ZM12 1.771c2.603 0 4.985.975 6.794 2.576-.182.26-1.731 2.33-5.375 3.696-1.68-3.084-3.54-5.61-3.827-6A10.424 10.424 0 0 1 12 1.77Zm-4.36.962c.273.365 2.095 2.903 3.8 5.922-4.79 1.276-9.02 1.25-9.475 1.25.664-3.176 2.812-5.818 5.675-7.172Zm-5.896 9.28v-.312c.443.013 5.414.078 10.53-1.458.299.573.572 1.158.832 1.744-.13.039-.273.078-.403.117-5.284 1.705-8.096 6.364-8.33 6.755a10.226 10.226 0 0 1-2.629-6.846ZM12 22.256c-2.369 0-4.555-.807-6.286-2.16.182-.378 2.264-4.387 8.043-6.404.026-.013.04-.013.065-.026 1.445 3.735 2.03 6.872 2.187 7.77-1.237.534-2.59.82-4.009.82Zm5.714-1.757c-.104-.625-.651-3.618-1.992-7.302 3.215-.507 6.026.326 6.378.443-.443 2.85-2.083 5.31-4.386 6.859Z" clip-rule="evenodd"></path>
  </g>
  <defs>
    <clipPath id="a">
      <path fill="#fff" d="M0 0h24v24H0z"></path>
    </clipPath>
  </defs>
</svg>"
`;

exports[`Test sample icons from each package > test AkMoon snapshot 1`] = `
"<svg width="1em" height="1em" fill="none" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
  <path stroke="currentColor" stroke-width="2" d="M20.958 15.325c.204-.486-.379-.9-.868-.684a7.684 7.684 0 0 1-3.101.648c-4.185 0-7.577-3.324-7.577-7.425a7.28 7.28 0 0 1 1.134-3.91c.284-.448-.057-1.068-.577-.936C5.96 4.041 3 7.613 3 11.862 3 16.909 7.175 21 12.326 21c3.9 0 7.24-2.345 8.632-5.675Z"></path>
  <path stroke="currentColor" stroke-width="2" d="M15.611 3.103c-.53-.354-1.162.278-.809.808l.63.945a2.332 2.332 0 0 1 0 2.588l-.63.945c-.353.53.28 1.162.81.808l.944-.63a2.332 2.332 0 0 1 2.588 0l.945.63c.53.354 1.162-.278.808-.808l-.63-.945a2.332 2.332 0 0 1 0-2.588l.63-.945c.354-.53-.278-1.162-.809-.808l-.944.63a2.332 2.332 0 0 1-2.588 0l-.945-.63Z"></path>
</svg>"
`;

exports[`Test sample icons from each package > test AkRibbon snapshot 1`] = `
"<svg width="1em" height="1em" fill="none" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
  <path stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M4 5v14.586c0 .89 1.077 1.337 1.707.707L12 14l6.293 6.293c.63.63 1.707.184 1.707-.707V5a2 2 0 0 0-2-2H6a2 2 0 0 0-2 2Z"></path>
</svg>"
`;

exports[`Test sample icons from each package > test AkWallet snapshot 1`] = `
"<svg width="1em" height="1em" fill="none" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
  <path stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M2 6a2 2 0 0 1 2-2h16a2 2 0 0 1 2 2v12a2 2 0 0 1-2 2H4a2 2 0 0 1-2-2V6ZM2 8h20"></path>
  <path stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M2 12h7c0 1 .6 3 3 3s3-2 3-3h7"></path>
</svg>"
`;

exports[`Test sample icons from each package > test CaAccessibilityColorFilled snapshot 1`] = `
"<svg width="1em" height="1em" viewBox="0 0 32 32" fill="currentColor" xmlns="http://www.w3.org/2000/svg">
  <path d="M16 10a8.598 8.598 0 0 0-7.965 6A8.598 8.598 0 0 0 16 22a8.598 8.598 0 0 0 7.965-6A8.598 8.598 0 0 0 16 10Zm0 10a4 4 0 1 1 4-4 4.005 4.005 0 0 1-4 4Z"></path>
  <path d="M16 2a14 14 0 1 0 14 14A14 14 0 0 0 16 2Zm0 22a10.654 10.654 0 0 1-9.97-7.758L5.97 16l.06-.242A10.654 10.654 0 0 1 16 8a10.654 10.654 0 0 1 9.97 7.758l.06.242-.06.242A10.654 10.654 0 0 1 16 24Z"></path>
  <circle cx="16" cy="16" r="2"></circle>
  <path d="M16 20a4 4 0 1 1 4-4 4.005 4.005 0 0 1-4 4Zm0-6a2 2 0 1 0 2 2 2.002 2.002 0 0 0-2-2Z" style="" fill="none"></path>
  <path d="M16 24a10.655 10.655 0 0 1-9.97-7.758L5.97 16l.06-.242A10.655 10.655 0 0 1 16 8a10.655 10.655 0 0 1 9.97 7.758l.06.242-.06.242A10.655 10.655 0 0 1 16 24Zm-7.965-8A8.598 8.598 0 0 0 16 22a8.598 8.598 0 0 0 7.965-6A8.598 8.598 0 0 0 16 10a8.598 8.598 0 0 0-7.965 6Z" style="" fill="none"></path>
  <path d="M0 0h32v32H0z" style="" fill="none"></path>
</svg>"
`;
